frontend/src/views/CensusDirectory/components/Project/ProjectButtons/components/EmbeddingButton/connect.ts

```diff
@@ -28,25 +25,46 @@ function pythonCodeSnippet(project: StaticProject | Project): string {
         census,
         organism = "${organism}",
         measurement_name = "${measurement}",
-        obs_value_filter = "tissue == 'tongue'",
-        obsm_layers = "${project.obs_matrix}"
+        obs_value_filter = "tissue_general == 'central nervous system'",
```

Comment:
why this magic string? Can we extract this string as a constant and provide some context please?

Reply:
It's just a static string, it's a question for @pablo-gar, but I think tongue was originally chosen because it was the smallest organ?